<?xml version="1.0" encoding="UTF-8"?>
<root BTCPP_format="4"
      main_tree_to_execute="MissionOrchestrator">
  <BehaviorTree ID="MissionOrchestrator">
    <SequenceWithMemory>
      <SubTree ID="PowerUp"/>
      <SubTree ID="ShutDown"/>
    </SequenceWithMemory>
  </BehaviorTree>

  <BehaviorTree ID="PowerUp">
    <AlwaysSuccess/>
  </BehaviorTree>

  <BehaviorTree ID="ShutDown">
    <AlwaysSuccess/>
  </BehaviorTree>

  <!-- Description of Node Models (used by Groot) -->
  <TreeNodesModel/>

</root>
